lldb/test/API/macosx/no-nlist-memory-module/Makefile

```diff
@@ -0,0 +1,19 @@
+C_SOURCES := main.c
+LD_EXTRAS = -Wl,-rpath "-Wl,$(shell pwd)" -L. -lno-nlists -lhas-nlists
+
+.PHONY: build-libno-nlists build-libhas-nlists
+all: build-libno-nlists build-libhas-nlists a.out
+
+include Makefile.rules
+
+build-libno-nlists: no-nlists.c no-nlist-sect.s
+	$(CC) $(CFLAGS) -c -o no-nlists.o $(<D)/no-nlists.c
+	$(CC) $(CFLAGS) -c -o no-nlist-sect.o $(<D)/no-nlist-sect.s
+	$(LD) -dynamiclib -o libno-nlists.dylib no-nlists.o no-nlist-sect.o -install_name "@executable_path/libno-nlists.dylib"
+
+build-libhas-nlists: has-nlists.c
+	$(CC) $(CFLAGS) -c -o has-nlists.o $(<D)/has-nlists.c
+	$(LD) -dynamiclib -o libhas-nlists.dylib has-nlists.o  -install_name "@executable_path/libhas-nlists.dylib"
+
+clean::
+	rm -rf has-nlists.o no-nlists.o no-nlist-sect.o main.o main.s a.out a.out.dSYM libno-nlists.dylib libno-nlists.dylib.dSYM libhas-nlists.dylib libhas-nlists.dylib.dSYM
```

Comment:
Would it be easier to `rm -rf *.o *.dSYM *.dylib`?

Reply:
yeah tbh I don't think we ever even run the clean targets these days, I just stuck this in there out of habit.  good suggestion.